core/src/main/scala/org/apache/spark/sql/rapids/tool/ToolUtils.scala

```diff
@@ -163,6 +163,9 @@ object ToolUtils extends Logging {
       values: Seq[String], fileDelimiter: String = QualOutputWriter.CSV_DELIMITER): String = {
     renderTextField(values, ":", fileDelimiter)
   }
+
+  var isMLOpsEnabled: Boolean = false
```

Comment:
this is a weird place/way to do this compared to the way all other args are handled.